tests/unit/test_alignment_score.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2023-present Kensho Technologies, LLC.
+import unittest
+
+from sequence_align.pairwise import alignment_score
+
+
+# Try something non-default
+DEFAULT_GAP = "?"
+
+
+class TestNWScore(unittest.TestCase):
```

Comment:
i think it would be a bit better to write out Needleman-Wunsch here instead of NW

Reply:
Similar thing -- remnant of when I had the function called `nw_score()` instead of `alignment_score()`. Will update